src/cdk/v2/destinations/zoho/rtWorkflow.yaml

```
@@ -12,7 +12,8 @@ steps:
   - name: validateConfig
     template: |
       const config = ^[0].destination.Config
-      $.assertConfig(config.region, "Datacentre Region is not present. Aborting")
+      const region = config.region ?? ^[0].destination.deliveryAccount.options.region
```

It should be the reverse
```suggestion
      const region = ^[0].destination.deliveryAccount.options.region ?? config.region
```